rust/actyx/ax/src/cmd/settings/local/get.rs

```diff
@@ -0,0 +1,61 @@
+use crate::cmd::{determine_ax_default_data_dir, settings::get::GetSettingsCommand, AxCliCommand};
+use ax_core::{
+    node::{self, lock_working_dir},
+    util::formats::{ActyxOSCode, ActyxOSError, ActyxOSResult},
+};
+use futures::{stream, Stream};
+use std::path::PathBuf;
+
+#[derive(clap::Parser, Debug, Clone)]
+pub struct SettingsLocalGetOpts {
+    #[command(flatten)]
+    actual_opts: GetSettingsCommand,
```

Comment:
Not a fan of the name, but I don't have a better suggestion

Reply:
That's the same name as the non-local variant of get/set opts.
And it is flattened so it won't show in the CLI prompt